### DISCUSSION TITLE
Set a title for the chart

### DISCUSSION THREAD

**Discussion by botradingblog1** [kernc/backtesting.py#768] [Q&A]
Hi,
First of all I 'love' backtest.py. Such an awesome backtester. Thanks so much!!! :)

I'm running backtester in batches for multiple stocks and it would be helpful if I could set the symbol for the HTML chart title or file name so I can differentiate between the different stocks.

Is there a way to do this?

Thanks!

Chris

**Comment by kernc** [comment]
Not presently without modifying plotting code. Opened https://github.com/kernc/backtesting.py/issues/769.

**Reply by botradingblog1** [reply]
Thanks for the quick response and the enhancement request. 
Chris
